CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ We'd like to offer a big thanks to the 3 contributors who made this release poss
 - 🐞 Bugfixes
 - 📚 Documentation improvements
 
+### Date Pickers
```

Comment:
In v6 alpha we used to put the breaking changes inside the package changes

```
### `@mui/x-date-pickers@6.0.0-alpha.0` / `@mui/x-date-pickers-pro@6.0.0-alpha.0`

#### Breaking changes
```

I'm not a fan of what has been done for alpha.0 here, the structure is redundant.

Reply:
Made a tentative change to the structure. 🤔 Let me know what you think